geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
         ol.geom.Polygon.fromCircle(featureGeom)
     );
   }
-  feature.set('name', 'element ' + (++this.featureSeq_));
+
+  /**
+   * @type {string}
+   */
+  var name;
+  switch (feature.getGeometry().getType()) {
+    case 'Point':
+      if (this.drawLabel.getActive()) {
+        name = this.translate_.getString('Label');
+      } else {
+        name = this.translate_.getString('Point');
+      }
+      break;
+    case 'LineString':
+      name = this.translate_.getString('LineString');
+      break;
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
+  feature.set('name', name + ' ' + (++this.featureSeq_));
+  this.drawnFeatures_.features.forEach(function(drawnFeature) {
+    if (feature.get('name') === drawnFeature.get('name')) {
+      feature.set('name', name + ' ' + (++this.featureSeq_));
+    }
+  }, this);
```

Comment:
What about initializing the sequence to the number of features + 1?
What I mean is that we could give the new feature a name with an index corresponding to the number of features already drawn.
```
feature.set('name', name + ' ' + this.drawnFeatures_.getCollection().getLength() + 1);
```
This is how google mymaps works.
Then, we can get rid of `featureSeq`.